server/src/config.ts

```diff
@@ -1,10 +1,29 @@
-/* tslint:disable:max-line-length */
+import { DbConfig } from "./models/index";
+
 export class Config {
   public static DB_USER: string = "postgres";
   public static DB_PW: string = "socobo";
-  public static DB_URL_TEST: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo_test?ssl=false`;
-  public static DB_URL_DEV: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo_dev?ssl=false`;
-  public static DB_URL: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo?ssl=false`;
+  public static DB_CONFIG_TEST: DbConfig = {
```

Comment:
Ok yes, I forgot that we use some of the values as env variables. I just wanted to access the db config object from inside the index.ts where the db is set up. Do we really need to set DB_CONFIG as env variable?